yet-another-docker-plugin/src/main/java/com/github/kostyasha/yad/launcher/DockerComputerSSHLauncher.java

```diff
@@ -65,6 +65,7 @@ public ComputerLauncher getPreparedLauncher(String cloudId, DockerSlaveTemplate
     @Override
     public void appendContainerConfig(DockerSlaveTemplate dockerSlaveTemplate, CreateContainerCmd createCmd) {
         final int sshPort = getSshConnector().port;
+        createCmd.withExposedPorts(new ExposedPort(sshPort));
```

Comment:
You need firstly get existing `ExposedPorts`, then check whether it already added and add new. Either launcher will reset any configured exposed ports before.